client/app/components/sections/Topbar/Topbar.js

```diff
@@ -156,15 +156,14 @@ class Topbar extends Component {
     const newListingRoute = this.props.routes && this.props.routes.new_listing_path ?
             this.props.routes.new_listing_path() :
             '#';
-
     const profileRoute = this.props.routes && this.props.routes.person_path && loggedInUsername ?
             this.props.routes.person_path(loggedInUsername) :
             null;
     const mobileMenuAvatarProps = this.props.avatarDropdown && loggedInUsername ?
             { ...this.props.avatarDropdown.avatar, ...{ url: profileRoute } } :
             null;
-    const isAdmin = this.props.avatarDropdown && loggedInUsername ?
-            this.props.avatarDropdown.isAdmin :
+    const isAdmin = this.props.isAdmin && loggedInUsername ?
+            this.props.isAdmin :
```

Comment:
this can be simplified: `const isAdmin = this.props.isAdmin && loggedInUsername`